Source/Manager/User/User+load.swift

```diff
@@ -14,4 +14,20 @@ public extension User {
         try? user.loadStoredTokens()
         return user
     }
+    /**
+     Enables App transfer step 1
+     */
+    static func storeOnDevice(withConfiguration clientConfiguration: ClientConfiguration, storageKey: String) {
+        let user = User(clientConfiguration: clientConfiguration)
+        try? user.storeOnDevice(key: storageKey)
+    }
+    /**
+     Enables App transfer step 2
+     */
+    static func loadFromDevice(withConfiguration clientConfiguration: ClientConfiguration, storageKey: String) -> User {
+        let user = User(clientConfiguration: clientConfiguration)
+        try? user.loadFromDeviceToKeychain(key: storageKey)
+
+        return User.loadLast(withConfiguration: clientConfiguration)
```

Comment:
We are creating an instance here, load something on it and then return from the object itself and not from the instance? 
Then why do we need the instance? 
Are we doing some static thing inside `loadFromDeviceToKeychain`

Reply:
aha I think I see it now👍

`return User.loadLast(withConfiguration: clientConfiguration)` is the new way, but we want to remove the old way too and move the stored data on UserDefautls to Keychain?

Reply:
Not sure what you mean. But the App transfer will constitue of two steps.

Step 1 (Code line 20): Storing keychain data to UserDefaults.
Step 2 (27): Storing UserDefaults data to keychain.

The code line: _return User.loadLast(withConfiguration: clientConfiguration)_
Perhaps could be omitted, and instead having the adopting sdk consumer call it when they want to.

Reply:
I am very new here so I have no info about the underlying code😁 let me read some more.

Now the question is why do we have to do it in two steps?🤔 why not just have both in one step? We are using UserDefault for something that is very sensitive. Probably you are gonna save it there for just a second, right? But many things can go wrong.

https://developer.apple.com/forums/thread/97231

Reply:
> I am very new here so I have no info about the underlying code😁 let me read some more.

Ah no worries. :D

> Now the question is why do we have to do it in two steps?🤔 why not just have both in one step? We are using UserDefault for something that is very sensitive. Probably you are gonna save it there for just a second, right? But many things can go wrong.
> 
> https://developer.apple.com/forums/thread/97231

There is a risk yes, although I think it is outweighed by the value of having users not being logged out, on doing an App transfer 🤔 . As the keychain gets cleaned out when doing an App transfer https://developer.apple.com/library/archive/technotes/tn2311/_index.html#//apple_ref/doc/uid/DTS40014135-CH1-A_ONE_TIME_LOSS_IN_KEYCHAIN_DATA_WILL_OCCUR_IF_YOU_SWITCH_YOUR_APP_ID_PREFIX

Reply:
Why do we need to call `User.loadLast(withConfiguration: clientConfiguration)` from here, isn't it enough to return `user` ?

Reply:
Thats right! Calling `User.loadLast(withConfiguration: clientConfiguration)`  is an unnecessary extra step. Omitting